man/systemd-boot.xml

```diff
@@ -444,6 +444,18 @@
       </varlistentry>
     </variablelist>
 
+    <para>The following <emphasis>UEFI Shell</emphasis> variables (vendor UUID <literal>158def5a-f656-419c-b027-7a3192c079d2</literal>)
+    are defined:</para>
+
+    <variablelist class='efi-variables'>
+      <varlistentry>
+        <term><varname>SYSTEMD_BOOT_LOG_LEVEL</varname></term>
+        <listitem><para>Logging level to use. One of <literal>fatal</literal>, <literal>error</literal>,
+        <literal>warning</literal> (default), <literal>info</literal>, <literal>debug</literal> or
+        <literal>trace</literal>.</para></listitem>
```

Comment:
Maybe mention how this can be set in the UEFI shell? (I know the latter commit adds a `bootctl` verb, but that only works from userspace.)